kolla/defaults.yml

```diff
@@ -310,9 +311,14 @@ enable_grafana: no
 enable_ceph: no
 
 # Container services (Zun and supporting)
-enable_etcd: no
-enable_kuryr: no
+enable_k3s: no
 enable_zun: no
+enable_zun_compute_k8s: "{{ enable_zun | bool and enable_k3s | bool }}"
+blazar_enable_device_plugin_k8s: "{{ enable_k3s | bool }}"
+# If not running K3s, can run standard Zun deploy w/ etcd and kuryr
+enable_etcd: "{{ enable_zun | bool and not enable_k3s | bool }}"
+enable_kuryr: "{{ enable_zun | bool and not enable_k3s | bool }}"
```

Comment:
We may not even want to imply that traditional Zun is deployable since we are not going to be testing it at all going forward.